### PR TITLE
pvoutput update template sensor example

### PR DESCRIPTION
**Pull request by SNoof85**
Update to new template sensor declaration

## Proposed change
Update to new template sensor declaration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/pvoutput.markdown [hunk 1]
```diff
@@ -49,23 +49,19 @@ sensor:
...
-        friendly_name: "Using"
-        unit_of_measurement: "Watt"
-      energy_consumption:
-        value_template: '{{ "%0.1f"|format(state_attr("sensor.pvoutput", "energy_consumption")|float/1000) }}'
-        friendly_name: "Used"
+template:
+  - sensor:
+    - name: power_consumption
+        state: "{% if is_state_attr('sensor.pvoutput', 'power_consumption', 'NaN') %}0{% else %}{{ state_attr('sensor.pvoutput', 'power_consumption') }}{% endif %}"
+        unit_of_measurement: "W"
```
Comment (frenck) [inline source/_integrations/pvoutput.markdown]: - Incorrect indentation
- Use normal names
- Template length (https://developers.home-assistant.io/docs/documenting/yaml-style-guide#template-string-length)

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by SNoof85** [comment]
Not sure about quotes here

File: source/_integrations/pvoutput.markdown [hunk 2]
```diff
@@ -49,24 +49,20 @@ sensor:
...
-        value_template: '{% if is_state_attr("sensor.pvoutput", "energy_generation", "NaN") %}0{% else %}{{ "%0.2f"|format(state_attr("sensor.pvoutput", "energy_generation")|float/1000) }}{% endif %}'
-        friendly_name: "Generated"
-        unit_of_measurement: "kWh"
+template:
+  - sensor:
+    - name: Power Consumption
+      state: "{{ state_attr('sensor.pvoutput', 'power_consumption') | float(default=0) }}"
+      unit_of_measurement: "W"
+    - name: Energy Consumption
+      state: "{{ "%0.1f"|format(state_attr('sensor.pvoutput', 'energy_consumption')|float(default=0) / 1000) }}"
```
Comment (frenck) [inline source/_integrations/pvoutput.markdown]:
```suggestion
      state: "{{ "%0.1f" | format(state_attr('sensor.pvoutput', 'energy_consumption') | float(default=0) / 1000) }}"
```

File: source/_integrations/pvoutput.markdown [hunk 3]
```diff
@@ -49,24 +49,20 @@ sensor:
...
+      state: "{{ state_attr('sensor.pvoutput', 'power_consumption') | float(default=0) }}"
+      unit_of_measurement: "W"
+    - name: Energy Consumption
+      state: "{{ "%0.1f" | format(state_attr('sensor.pvoutput', 'energy_consumption') | float(default=0) / 1000) }}"
+      unit_of_measurement: "kWh"
+    - name: Power Generation
+      state: "{{ state_attr('sensor.pvoutput', 'power_generation') | float(default=0) }}"
+      unit_of_measurement: "W"
+    - name: Energy Generation
+      state: "{{ "%0.2f"|format(state_attr('sensor.pvoutput', 'energy_generation') | float(default=0) / 1000) }}"
```
Comment (frenck) [inline source/_integrations/pvoutput.markdown]:
```suggestion
      state: "{{ '%0.2f' | format(state_attr('sensor.pvoutput', 'energy_generation') | float(default=0) / 1000) }}" 
```

**Review by frenck — APPROVED** [review]
Small tweak, but other than that, awesome!

Thanks @SNoof85 👍 

(PS: I've rebased it to deal with the CI issue).

**Comment by SNoof85** [comment]
Awesome. Thx @frenck

**Pull request merged**